src/main/kotlin/com/toasttab/pulseman/view/AuthSelectorUI.kt

```diff
@@ -15,8 +15,6 @@
 
 package com.toasttab.pulseman.view
 
-import androidx.compose.desktop.DesktopTheme
-import androidx.compose.desktop.SwingPanel
```

Comment:
Most of the following UIs just have state hoisting implemented and removing uneeded things like MaterialTheme.
In general the views have been simplified a lot